includes/gateways/stripe/includes/payment-methods/class-give-stripe-becs.php

```diff
@@ -16,22 +16,22 @@
 /**
  * Check for Give_Stripe_Becs existence.
  *
- * @since 2.6.3
+ * @since 2.7.0
```

Comment:
If this has existed since 2.6.3, then this shouldn't be updated.

Reply:
@bradp This BECS Direct Debit related functionality is not released and will get released as a part of `2.7.0`. So, I have updated it to avoid confusion in the future.